tracer/contrib/net/httptrace/example_test.go

```diff
@@ -0,0 +1,24 @@
+package httptrace_test
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/DataDog/dd-trace-go/tracer"
+	"github.com/DataDog/dd-trace-go/tracer/contrib/net/httptrace"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	span := tracer.SpanFromContextDefault(r.Context())
+	fmt.Printf("tracing service:%s resource:%s", span.Service, span.Resource)
+	w.Write([]byte("hello world"))
+}
+
+func Example() {
+	mux := http.NewServeMux()
+	mux.Handle("/users", handler)
+	mux.Handle("/anything", handler)
```

Comment:
`handler` can't be used as a http.Handler, use `http.HandlerFunc(handler)` it will create an http.Handler struct